qi-lib/flow/core/deforest.rkt

```diff
@@ -420,7 +426,7 @@
                  [countdown init-countdown])
         ((next (λ () ((contract (-> pair? any)
                                 (λ (v) v)
-                                'cadr-cstream-next ctx #f
+                                'cad*r-cstream-next ctx #f
```

Comment:
This typo fix should be part of the previous commit where the typo was
introduced (so that we never see the typo + fix as part of the commit history).